src/runtime-rs/crates/hypervisor/src/device/driver/virtio_fs.rs

```diff
@@ -42,14 +49,12 @@ pub struct ShareFsMountConfig {
     pub prefetch_list_path: Option<String>,
 }
 
-#[derive(Debug, Clone)]
-pub struct ShareFsMountDevice {
-    pub config: ShareFsMountConfig,
-}
-
 /// ShareFsDeviceConfig: share fs device config
-#[derive(Debug, Clone)]
-pub struct ShareFsDeviceConfig {
+#[derive(Clone, Debug, Default)]
+pub struct ShareFsConfig {
+    /// shared_path: the upperdir of the passthrough fs exported dir or rafs meta file of rafs
+    pub shared_path: String,
```

Comment:
Is `shared_path` a rename of what was previously `host_path`?

Reply:
Yes, IMO, `shared_path` is more accurate than `host_path`, which comes from `get_host_ro_shared_path`.

Reply:
@Apokleos I would prefer to keep this as `host_path` as it makes it clearer or maybe even `host_shared_path`.

Reply:
`host_share_path` :+1:

Reply:
done. host_shared_path 👍